server/routes/middleware/requiresAuth.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require("jsonwebtoken");
+const { createErrorResponse } = require("./util");
+
+module.exports = async (req, res, next) => {
+  const { token } = req.cookies;
+
+  if (!token) {
+    return createErrorResponse(res, 403, { message: "Forbidden" });
+  } else {
```

Comment:
Return 401 here too

Reply:
should I replace 403 with 401?

Reply:
update: i replaced it. I understand why it should be 401 now